Sources/Extensions/AppIntents/Widget/Gauge/WidgetGaugeAppIntent.swift

```diff
@@ -136,6 +182,7 @@ struct WidgetGaugeAppIntent: WidgetConfigurationIntent {
 @available(iOS 17.0, macOS 14.0, watchOS 10.0, *)
 enum GaugeTypeAppEnum: String, Codable, Sendable, AppEnum {
     case normal
+    case withoutMinMax
```

Comment:
Can you rename it to `singleLabel`?
I think this way it's clearer, one type shouldn't impact the name of the following one

Reply:
yup makes sense. lmk if you want to change the display name in the UI as well

Reply:
Perhaps can you add a comment for each one to describe what they are meant to? I was reading again the code and for someone that will jump to this file in the future, there are so many "labels". So... something like:

// Label X is meant for this
// Label Y is for that

Reply:
added